devbox.go

```diff
@@ -27,11 +27,12 @@ type Devbox interface {
 	Generate() error
 	GenerateDevcontainer(force bool) error
 	GenerateDockerfile(force bool) error
-	GenerateEnvrc(force bool, source string) error
+	CreateEnvrcFile(force bool) error
```

Comment:
Should `GenerateDevcontainer` and `GenerateDockerfile` and `CreateEnvrcFile` use the same verb?